test/interface/fitpredict.jl

```diff
@@ -37,7 +37,7 @@ end
     ŷ = MLJBase.predict(lr, fr, Xt)
     ŷ = MLJBase.mode.(ŷ)
 
-    mcr = MLJBase.misclassification_rate(ŷ, yc)
+    mcr = MLJ.misclassification_rate(ŷ, yc)
```

Comment:
I doubt you really need MLJ here. The metrics used to be defined in MLJBase, but now they are provided by StatisticalMeasures.jl